scripts/ci-checks.sh

```diff
@@ -57,7 +57,7 @@ if [ ! -f "scripts/env/bin/activate" ]
 fi
 
 source scripts/env/bin/activate
-pip --disable-pip-version-check install -U black pylint mypy 1>/dev/null
+pip --disable-pip-version-check install -U black pylint "mypy<0.900" 1>/dev/null
```

Comment:
What's this pinned version fixing, and is it resolved by `mypy 0.901`?

Reply:
Not so much fixing as working around https://github.com/python/mypy/issues/10596. I haven't tried 0.901 but hopefully yes :)